container-engine-lib/lib/backend_impls/docker/docker_kurtosis_backend/docker_kurtosis_backend_enclave_functions.go

```diff
@@ -24,6 +25,10 @@ const (
 
 	defaultHttpLogsCollectorPortNum = uint16(9712)
 	defaultTcpLogsCollectorPortNum  = uint16(9713)
+
+	labelTypeName       = "com.kurtosistech.container-type"
```

Comment:
I suggest you use this constant instead `label_key_consts.containerTypeLabelKeyStr` you will have to make it public but if it's modified in the future won't affect your logic here.